test/markup/xml/document-type-variations.expect.txt

```diff
@@ -0,0 +1,38 @@
+<span class="hljs-meta">&lt;!DOCTYPE<span class="hljs-meta-keyword"> svg</span><span class="hljs-meta-keyword"> PUBLIC</span> <span class="hljs-meta-string">" -//W3C//DTD SVG 1.1 Basic//EN "</span> <span class="hljs-meta-string">' http://www.w3.org/Graphics/SVG/1.1/DTD/svg11-basic.dtd '</span> &gt;</span>
+<span class="hljs-meta">&lt;!DOCTYPE<span class="hljs-meta-keyword"> svg</span><span class="hljs-meta-keyword"> PUBLIC</span><span class="hljs-meta-string">"-//W3C//DTD SVG 1.1 Basic//EN"</span><span class="hljs-meta-string">'http://www.w3.org/Graphics/SVG/1.1/DTD/svg11-basic.dtd'</span>&gt;</span>
+
+<span class="hljs-meta">&lt;!DOCTYPE<span class="hljs-meta-keyword"> note</span> [
+<span class="hljs-meta">&lt;!ENTITY<span class="hljs-meta-keyword"> nbsp</span> <span class="hljs-meta-string">'&amp;#xA0;'</span>&gt;</span>
+<span class="hljs-meta">&lt;!ENTITY<span class="hljs-meta-keyword"> pound</span> <span class="hljs-meta-string">'&amp;#xA3;'</span>&gt;</span>
+<span class="hljs-meta">&lt;!ENTITY<span class="hljs-meta-keyword"> writer_person</span> <span class="hljs-meta-string">" Writer: Donald Duck. "</span>&gt;</span>
+<span class="hljs-meta">&lt;!ENTITY<span class="hljs-meta-keyword"> copyright</span> <span class="hljs-meta-string">"Copyright: Walt Disney Company."</span>&gt;</span>
```

Comment:
Why aren't the `"` converted to `&quot;` here? (like in the prior expect) Am I missing something or have we found a bug?

Reply:
I messed up the first solution. I inserted the code generated by the php htmlentities () function instead of the result of the hljs highlighting.

That is why the first solution code was bad and contained &quot; entities.